docs/jwst/charge_migration/description.rst

```diff
@@ -0,0 +1,52 @@
+Description
+===========
+
+:Class: `jwst.charge_migration.ChargeMigrationStep`
+:Alias: charge_migration
+
+
+Overview
+--------
+This step corrects for an artifact seen in undersampled NIRISS images that may depress flux 
+in resampled images. The artifact is seen in dithered images where the star is centered in 
+a pixel. When the peak pixels of such stars approach the saturation level, they suffer from 
+significant :ref:`charge migration <charge_migration>`:
+the spilling of charge from a saturated pixel into its neighboring pixels. This charge migration 
+causes group-to-group differences to decrease significantly once the signal level is greater than 
+~30,000 ADU. As a result, the last several groups of these ramps get flagged by the ``jump`` step. 
```

Comment:
Could make the references to the jump and ramp_fitting steps actual links to those step docs. Oh, and of course the 30,000 value will need to be modified to 25,000 (as in the other PR).

Reply:
Done